src/mlpack/methods/kmeans/naive_kmeans_impl.hpp

```diff
@@ -39,47 +39,44 @@ double NaiveKMeans<DistanceType, MatType>::Iterate(const arma::mat& centroids,
   counts.zeros(centroids.n_cols);
 
   // Find the closest centroid to each point and update the new centroids.
-  // Computed in parallel over the complete dataset
   #pragma omp parallel
   {
-    // The current state of the K-means is private for each thread
-    arma::mat localCentroids(centroids.n_rows, centroids.n_cols,
-        arma::fill::zeros);
-    arma::Col<size_t> localCounts(centroids.n_cols, arma::fill::zeros);
+    // Thread-local storage for partial sums
+    arma::mat threadCentroids(centroids.n_rows, centroids.n_cols, arma::fill::zeros);
+    arma::Col<size_t> threadCounts(centroids.n_cols, arma::fill::zeros);
```

Comment:
These changes don't actually change anything. Can you revert any changes where there is no actual functionality change?  The only change here is a change to the name of the variables, and I don't see any reason to do that (plus it makes the diffs across history unnecessarily larger).